source/settings/plugin/fts-plugin.rst

```diff
@@ -300,6 +300,44 @@ See also :ref:`fts_tokenization`
 
 .. _plugin-fts-setting-fts_index_timeout:
 
+``fts_headers_blacklist`` , ``fts_headers_whitelist``
+-----------------------------------------------------
+
+.. versionadded:: 2.3.18
+
+- Default: <empty>
+- Values:  :ref:`string`
+
+The list of headers to, respectively, whitelist or blacklist. Whitelist takes
+precedence over blacklist: if a header matches both, it is indexed. The default
+is the former behavior, i.e. index all headers.
+
+The terms are case insensitive. An asterisk ``*`` at the end of the header name
+matches anything starting with that header name. The asterisk can only be used
+at the end of the header name, prefix and infix usage are not supported.
+
+Example::
+
+  plugin {
+    fts_headers_blacklist = Received DKIM-* X-* ARC-* DomainKey* Authentication-Results Comments
+    fts_headers_whitelist = From To Cc Bcc Subject Message-ID In-Reply-To References Keywords Comments
+  }
+
+All ``X-`` experimental headers, all ``DKIM-`` headers and all headers
+starting  with ``DomainKey`` are blacklisted, along with
+``Authentication-Results``. ``Keywords`` is indexed anyway, as it appears
+**both** in *blacklist* and *whitelist*: whitelist takes precedence.
```

Comment:
You mean "Comments", not "Keywords".

Anyway, the examples could be a bit more "real world" use cases. Now I think in the above one the whitelist one is basically ignored, since the default behavior is to index those headers anyway? So I was thinking the examples could be:
 * One example where most headers are excluded except a few (this is really the example you already have below)
 * One example where most header are included except a few. So this could be the above example, except in whitelist have only e.g. "X-Spam-Status" to indicate that it overrides the X-* exclude.